### PR TITLE
#24

### PR DESCRIPTION
**Pull request by AgnieszkaKurek**
Please review this change and wrte me if it is safe to merge it to master.

### CODE REVIEW

File: scripts/slider.js [hunk 1]
```diff
@@ -17,8 +17,11 @@ function hide() {
 }
 
 function changeSlide() {
-    number++; if (number > 22) number = 1;
-    var file =`<img src="../images/slides/slide${number}.jpg"/>` ;
+    number++;
+    if (number > 22) {
+        number = 1;
+    };
```
Comment (Krzysiek102) [inline scripts/slider.js:23]: Is this semicolon needed there?

**Review by Krzysiek102 — CHANGES_REQUESTED** [review]
In general looks ok.

File: scripts/slider.js (hunk 1 above)
Comment (AgnieszkaKurek) [reply scripts/slider.js:23]: Yes, it is needed.

Comment (Krzysiek102) [reply scripts/slider.js:23]: Ok

**Review by Krzysiek102 — APPROVED** [review]

**Pull request merged**